frontend/src/App.tsx

```diff
@@ -1,17 +1,31 @@
-import React from "react";
-import { Route, Routes } from "react-router-dom";
 
 import { Container, Footer, Navigation } from "hds-react";
-
-import CompanyListing from "./pages/CompanyListing";
-import Apartmentslisting from "./pages/ApartmentsListing";
-import Reports from "./pages/Reports";
-import Documents from "./pages/Documents";
-import Codes from "./pages/Codes";
+import { Outlet } from "react-router-dom";
 
 import "./styles/index.sass";
 
-function App(children) {
+function App() {
+    // Function for formatting the NavItem titles into label texts
+    const formatStr = (str: string): string => {
+        if (str === null) return "";
+        const strArray =
+            str && str === "companies" // make yhtiot (for url) into yhtiöt, for the label string
+                ? "yhtiöt".split("")
+                : str.split("");
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        const firstLetter = strArray.shift().toUpperCase();
+        return firstLetter + strArray.join("");
+    };
```

Comment:
Might be a it simpler like this:
`string.charAt(0).toUpperCase() + string.slice(1);`